### PR TITLE
Chore_humans_txt_update

### PR DESCRIPTION
**Pull request by andykwoods**
## Description

Humans.txt file was not up to date with the current prod+eng department. 

Fixes #
-Added in new team members
-Moved leavers to the Thanks section
-Sorted the list alphabetically. 

## Checklist

- [ ] Approval

### CODE REVIEW

**Review by jimCresswell — APPROVED** [review]

File: public/humans.txt [hunk 1]
```diff
@@ -5,76 +5,88 @@ Oak National Academy’s tech platform has been created by a team of human devel
...
+  Amy Matthews (Infrastructure Engineer)
+  From: Kent, UK
+
+  Andrew Woods (Software Engineering Manager)
+  Site: https://andrewwoods.super.site/
+  LinkedIn: https://www.linkedin.com/in/andykwoods/
+  From: Dublin, Ireland
+
+  Ben Protheroe (Software Engineer)
+  From London, lives in Cornwall
```
Comment (jack-skerrett-oak) [inline public/humans.txt:17]: missing colon

File: public/humans.txt [hunk 2]
```diff
@@ -5,76 +5,88 @@ Oak National Academy’s tech platform has been created by a team of human devel
...
+
   Ian Kynnersley (Head of Engineering)
   Site: https://www.iankynnersley.co.uk/
   LinkedIn: https://www.linkedin.com/in/iankynnersley/
   Twitter: https://twitter.com/kpopper
   From: London, UK
   
-
-  Jack Skerrett (QA)
+  Jack Skerrett (Quality Assurance Engineer)
```
Comment (jack-skerrett-oak) [inline public/humans.txt:37]: My official title is "Quality Assurance Analyst"

File: public/humans.txt [hunk 3]
```diff
@@ -5,76 +5,88 @@ Oak National Academy’s tech platform has been created by a team of human devel
...
-
-  Matt Gregory (Developer)
+  Matt Gregory (Software Engineer)
   Site: https://mantagen.org
 
-
-  Ross Mackay (Developer)
+  Monika Koziol (Principal Product Designer)
+  Linkedin: https://uk.linkedin.com/in/monika-koziol-a138a325
+  From: Poland lives in London
```
Comment (jack-skerrett-oak) [inline public/humans.txt:65]: Missing comma

File: public/humans.txt (hunk 2 above)
Comment (andykwoods) [reply public/humans.txt]: updated

File: public/humans.txt (hunk 3 above)
Comment (andykwoods) [reply public/humans.txt]: updated

File: public/humans.txt [hunk 4]
```diff
@@ -5,75 +5,87 @@ Oak National Academy’s tech platform has been created by a team of human devel
...
   From: UK
 
-
-  Joe Baker (Developer)
+  Joe Baker (Software Engineer)
   LinkedIn: https://www.linkedin.com/in/joe-baker-96a109219/
   From: London, UK
 
+  Ben Protheroe (Software Engineer)
+  From London, lives in Cornwall
```
Comment (johnrobeds) [inline public/humans.txt]: Needs a :

Comment (andykwoods) [reply public/humans.txt]: that is fixed in this commit
https://github.com/oaknational/Oak-Web-Application/pull/1659/commits/12786b229b677336cf6e632e8308d27e01b06d07

**Pull request merged**